### PR TITLE
[15.0][MIG] stock_picking_delivery_link

### PR DESCRIPTION
**Pull request by flachica**

### CODE REVIEW

File: stock_picking_delivery_link/tests/common.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Copyright 2020-2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests import SavepointCase
+
+
+class StockPickingDeliveryLinkCommonCase(SavepointCase):
```
Comment (victoralmau) [inline stock_picking_delivery_link/tests/common.py]:
```suggestion
from odoo.tests import TransactionCase


class StockPickingDeliveryLinkCommonCase(TransactionCase):
```

File: stock_picking_delivery_link/tests/common.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        Move = self.env["stock.move"]
+        # simulate create + onchange
+        move = Move.new(
+            {
+                "product_id": product.id,
+                "location_id": src_location.id,
+                "location_dest_id": dst_location.id,
+            }
+        )
+        move.onchange_product_id()
```
Comment (victoralmau) [inline stock_picking_delivery_link/tests/common.py]:
```suggestion
        move._onchange_product_id()
```

**Review by victoralmau — CHANGES_REQUESTED** [review]

**Comment by flachica** [comment]
Thanks @victoralmau I was distracted. I am doing a migration of a module with many dependencies and I am gradually checking

**Review by victoralmau — APPROVED** [review]
Code review OK (although I have never used this module).

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**